text/0000-roadmap-2020.md

```diff
@@ -0,0 +1,367 @@
+
+Part of the problem here is money. One of the biggest challenges
+around organizational work is that it is quite demanding in terms of
+time. It requires availability. It is difficult to do in your spare
+time. Therefore, helping to ensure that it is easier for people to get
+paid for their work on Rust -- and especially their **organizational**
+work -- is one way we might make progress here.
+
+However, it's worth emphasizing that this doesn't necessarily mean
+people whose job description is *solely* to work on Rust. There are
+many companies using Rust, and many of them would like to help out,
+but we need to do better at harnessing and directing those efforts.
+As Parity put it in their #rust2020 post:
+
+> “We, too, have team members who are interested in helping on
+> specialization or fixing the aforementioned bugs. However, it’s
+> often unclear whether the work is worthwhile. To a business, it is
+> hard to argue that one might spend a month or two working on a new
+> feature without any assurance that the approach taken would be
+> accepted.”
+>
+> -- [Benjamin Kampmann, speaking for Parity](https://www.parity.io/rust-2020/)
+
+### Make design discussions more productive and less exhausting
+
+> An RFC, or "request for comments" is a mechanism by which a group of
+> people can get feedback from a wider community on proposed
+> changes. The idea is that a written proposal outlines a change's
+> scope, implementation details, rationale and impact on the
+> ecosystem, then people make comments on the proposal. Usually by the
+> time that everybody has stopped shouting at each other, the RFC is
+> ready to be merged, meaning it is accepted and its vision can be
+> implemented. This can either be implementing a feature, or removing
+> unstable flags from it.
+>
+> -- [spacekookie](https://spacekookie.de/blog/rust-2020-the-rfc-process-and-distributions/)
+
+The RFC process has been a crucial part of Rust's organization for a
+long time. The process of documenting and talking over our designs is
+often very helpful for improving the design and sometimes leads to
+dramatic changes. Many other languages have adopted RFCs and
+explicitly cited Rust as precedent.
+
+Of course, we also have ample evidence that the RFC process as
+presently practiced does not work well for larger-scale or
+controversial designs. Last year we put a lot of energy into thinking
+about techniques for improving the process, and this year we need to
+put more of that energy into actually making those changes.
+
+# Yearly calendar
+
+Here is a rough calendar of major events in the planning of Rust. Note
+that we have attempted to move up some of the Rust 2021 planning --
+e.g., the survey, edition, and so forth -- so that they begin earlier
+in 2020, versus the timing from this year.
+
+* January
+  * Rust 2019 survey results published
+  * Roadmap RFC opened
+* February 
+* March
+  * Rust All Hands will take place March 16-20
+  * Publish All Hands retrospective
```

Comment:
This should probably be updated since All Hands was cancelled (or postponed? unclear)